apps/web/src/components/GetStarted/Funding.tsx

```diff
@@ -0,0 +1,41 @@
+import { sectionContainerClasses, titleClasses } from '../../../app/(base-org)/getstarted/page';
+import ResourceGrid from './ResourceGrid/ResourceGrid';
+import { ResourceSectionType } from './resourceTypes';
+
+const fundingResources: ResourceSectionType = {
+  title: 'Fund your project',
+  colorOne: 'green-80',
+  colorTwo: 'green-60',
+  cards: [
+    {
+      title: 'Rounds Grants',
+      description:
+        'Post to /base-builds on Farcaster to be eligible for 2.25 ETH in weekly rewards ',
+      href: 'https://warpcast.com/base/0xb3f1428b?utm_source=dotorg&urm_medium=builderkit',
+    },
+    {
+      title: 'Gas Credits',
+      description: 'Eligible projects may receive up to $15K in gas credits for their users',
+      href: 'https://www.smartwallet.dev/base-gasless-campaign/?utm_source=dotorg&utm_medium=builderkit',
+    },
+    {
+      title: 'Base Builder Grants',
+      description:
+        'Apply for a retroactive Base Builder Grant, rewarding great projects being built on Base',
+      href: 'https://paragraph.xyz/@grants.base.eth/calling-based-builders/?utm_source=dotorg&utm_medium=builderkit',
+    },
+  ],
+};
+
+export const FUNDING_SECTION_ID = 'GetFunded';
+
+export default async function Funding() {
+  return (
+    <div id={FUNDING_SECTION_ID} className={sectionContainerClasses}>
```

Comment:
Yeah this could be a component called SectionContainer

Reply:
opted not to create a bespoke container in favor of the shared styles approach, but let me know if you'd prefer a component